scripts/process/process.order_sites.R

```diff
@@ -4,25 +4,10 @@ process.order_sites <- function(viz = as.viz("order-sites")){
   
   depends <- readDepends(viz)
   
-  sites <- depends[["nwis-sites"]]
+  sites <- as.data.frame(depends[["sp-sites"]])
   
   reordered_sites <- sites %>%
-    arrange(desc(dec_long_va)) 
-  
-  reordered_sites$station_nm <- sapply(reordered_sites$station_nm, siteCapper)
+    arrange(desc(coords.x1)) 
```

Comment:
I think this should be `arrange(coords.x1)` instead. The way it currently is set up puts the earliest rivers on top, and I think it feels more natural to have them on the bottom.